examples/gallery/maps/country_polygons.py

```diff
@@ -0,0 +1,98 @@
+"""
+Highlight country, continent and state polygons
+-----------------------------------------------
+The :meth:`pygmt.Figure.coast` method can highlight country polygons
+via the ``dcw`` parameter. It accepts the country code or full
+country name and can draw its borders and add a color to its landmass.
+It's also possible to define multiple countries at once by separating
+the indiviudal names with commas.
+"""
+
+# sphinx_gallery_thumbnail_number = 1
+
+
+import pygmt
+
+fig = pygmt.Figure()
+
+fig.coast(
+    region=[-12, 32, 34, 72],
+    # Lambert Azimuthal Equal Area lon0/lat0/horizon/width
+    projection="A10/52/25/6c",
```

Comment:
Using this projection makes sense.

I am wondering whether we want to show Spitsbergen?
(Or was it actively excluded by using `52`° N for `lat0` instead of `53`° N?)

```suggestion
    region=[-12, 32, 34, 80],
    # Lambert Azimuthal Equal Area lon0/lat0/horizon/width
    projection="A10/57/25/6c",
```

![coast_dcw_spitsbergen](https://github.com/GenericMappingTools/pygmt/assets/94163266/24f7bffe-f448-4195-b840-4074bca2b6f9)

Reply:
The 52° N for lat0 is just the Lambert Azimuthal Equal Area's center of projection, and I suggested it at https://github.com/GenericMappingTools/pygmt/pull/2428#discussion_r1309408465 to follow [ETRS89-LAEA](https://en.wikipedia.org/wiki/European_grid)/[EPSG:3035](https://epsg.io/3035). But we can shift the projection centre Northward also.

Reply:
I would leave it as is since otherwise there's so much water area without related content visible.

Reply:
Joa, just realized yesterday evening that I excluded Spitsbergen in my code suggestion :sweat_smile:. So, we can leave it as is if the others are fine with it.

Reply:
Ok, let's leave it as is, I'll merge this PR in now.